samples/bluetooth/mesh/light/src/model_handler.c

```diff
@@ -69,14 +69,18 @@ static void led_set(struct bt_mesh_onoff_srv *srv, struct bt_mesh_msg_ctx *ctx,
 	}
 
 	led->value = set->on_off;
+	if (!bt_mesh_model_transition_time(set->transition)) {
+		led->remaining = 0;
+		dk_set_led(led_idx, set->on_off);
+		goto respond;
+	}
+
```

Comment:
IMO no, as we'll get a pyramid of scopes, ending at 4 indentation levels. 

Zephyr code tries to keep the "happy path" on the innermost indentation level whenever possible, since this makes the error handling stand out, letting the reader ignore it. I think this works very well, and would like to keep this in our code as well :slightly_smiling_face: